cmds/xbmc.py

```diff
@@ -664,29 +675,27 @@ def _files(bot,args,dirs,pid):
   if len(matches)==0:
     return 'Found 0 matches'
 
-  # default to top dir if every match is a sub dir
-  matches = util.reducetree(matches)
-
-  if len(matches)>1:
+  if len(matches)>1 and playall==False:
     maxm = bot.opt('library.max_matches')
     if maxm<1 or len(matches)<=maxm:
       return 'Found '+str(len(matches))+' matches: '+util.list2str(matches)
     else:
       return 'Found '+str(len(matches))+' matches'
 
   # translate library path if necessary
-  match = bot.library_translate(matches[0])
+  #match = bot.library_translate(matches[0]) 
```

Comment:
Ahh okay, did not know this. I'm still looking into making the library_translate() work with lists, but in the meantime, this can be reverted to avoid any bad mojo